po/extra/fr.po

```diff
@@ -0,0 +1,72 @@
+# French translations for extra package.
+# Copyright (C) 2019 THE extra'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the extra package.
+# NathanBnm, 2019.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: extra\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2019-06-22 16:07+0200\n"
+"PO-Revision-Date: 2019-06-22 16:01+0200\n"
+"Last-Translator: NathaBnm\n"
+"Language-Team: none\n"
+"Language: fr\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+
+#: data/com.github.ryonakano.spreadsheet.desktop.in:3
+#: data/com.github.ryonakano.spreadsheet.appdata.xml.in:7
+msgid "Spreadsheet"
+msgstr "Spreadsheet"
```

Comment:
I don't really know.
In French it will looks like "Classeur" or "Feuille de calcul" and in my opinion this is a little weird for an app name, si I don't know about this point.